resources/workflow/invalid_measures/unloadable_measure.osw

```diff
@@ -0,0 +1,19 @@
+{
+	"weather_file": "../../Examples/compact_osw/files/srrl_2013_amy.epw",
+	"seed_file": "../example_model.osm",
+	"measure_paths": ["../measures/"],
+	"steps": [
+		{
+			"measure_dir_name": "FakeModelMeasure",
+			"arguments": {}
+		},
+		{
+			"measure_dir_name": "UnloadableMeasure",
+			"arguments": {}
```

Comment:
This is an existing directory (findMeasure works), but it doesn't have a measure.xml